lib/reference/atKeywords.js

```diff
@@ -2,6 +2,9 @@
 
 const uniteSets = require('../utils/uniteSets.js');
 
+// https://www.w3.org/TR/css-nesting-1/#conditionals
+const nestingSupportedAtKeywords = new Set(['container', 'layer', 'media', 'scope', 'supports']);
```

Comment:
This is different from `pageMarginAtKeywords`.

These rules or their grouping is not defined by nesting.
They each have their own definition but happen to have extra allowances in nesting.

There might be a future groupings with only `media, scope, container` or `media, supports, when, else`.

But `atKeywords` must always contain all known at keywords.
Even if a utility grouping is altered or removed.

----

I made it a separate Set and did not include it in the list of `uniteSets`.

Instead I added a unit test to make sure that all items in `nestingSupportedAtKeywords` also exist in `atKeywords`.

Thoughts?

Reply:
Sounds good 👍🏼